python/svix/webhooks.py

```diff
@@ -29,6 +29,16 @@ def __init__(self, whsecret: str, *, enc_key: t.Optional[str] = None):
         self._whsecret = base64.b64decode(whsecret)
         self._enc_key = base64.b64decode(enc_key) if enc_key is not None else None
 
+    def from_raw(whsecret: bytes):
+        wh = Webhook()
+        wh._whsecret = whsecret
+        return wh
+
+    def from_raw(whsecret: str):
+        wh = Webhook()
+        wh._whsecret = str.encode(whsecret)
```

Comment:
```suggestion
        wh._whsecret = whsecret.encode()
```
I think that's the python way of doing it.